src/material/schematics/ng-update/migrations/legacy-components-v15/constants.ts

```diff
@@ -15,9 +15,8 @@ export const COMPONENTS = [
   'dialog',
   'form-field',
   'input',
+  'list',
   'menu',
-  'option',
```

Comment:
should `core` be added instead of these?

Reply:
Oh I moved these out to explicitly only add them for mixins on line 43. I added core there, too. Lmk if it looks ok now

Reply:
I think core should be here though, since you can do a ts `import {...} from '@angular/material/core'`

Reply:
Oh, I didn't know that lol. Fixed!